zerver/webhooks/jira/view.py

```diff
@@ -369,8 +418,16 @@ def api_jira_webhook(
     if content_func is None:
         raise UnsupportedWebhookEventTypeError(event)
 
+    auth = None
+
+    if email and jira_api_token is not None:
+        # Config options encode parameters using Jira's internal encoding,
```

Comment:
This comment makes no sense as config options are part of Zulip's Webhook integration code ...